x-pack/docs/en/security/backup-security-configuration.asciidoc

```diff
@@ -0,0 +1,263 @@
+location on a cloud storage service or on a shared network file system. This is
+used to snapshot the `.security` index. The snapshot operation can also
+optionally capture the persistent settings from the global cluster metadata.
+
+. Create a repository that you can use to backup the `.security` index.
+Ideally this repository is dedicated only to this scope.
++
+--
+[source,js]
+-----------------------------------
+PUT /_snapshot/my_backup
+{
+  "type": "fs",
+  "settings": {
+    "location": "my_backup_location"
+  }
+}
+-----------------------------------
+// CONSOLE
+// TESTSETUP
+
+You must have the elevated `manage` cluster privilege to create it, yet it's a
+relatively infrequent operation. The elevated privileges are required to prevent
+regular users from exfiltrating data to locations of their choosing.
+
+Afterwards, users with the limited `snapshot_user` built-in role can create
+snapshots inside that repository. This role grants privileges to create
+snapshots in any existing repositories. It also grants privileges to list and
+read settings for all indices (but not to read data).
+
+For more information, see {stack-ov}/security-privileges.html[Security privileges]
+and {stack-ov}/built-in-roles.html[Built-in roles].
+--
+
+. Create users that have the `snapshot_user` built-in role.
++
+--
+The following example creates a new user `snapshot_user` in the native realm:
+
+[source,js]
+--------------------------------------------------
+POST /_security/user/snapshot_user
+{
+  "password" : "secret",
+  "roles" : [ "snapshot_user" ]
+}
+--------------------------------------------------
+// CONSOLE
+--
+
+. Create a snapshot.
++
+--
+The following example exemplifies the create snapshot operation, inside the
+`my_backup` repository:
+
+[source,js]
+--------------------------------------------------
+PUT /_snapshot/my_backup/snapshot_1
+{
+  "indices": ".security",
+  "include_global_state": true <1>
+}
+--------------------------------------------------
+// CONSOLE
+
+<1> A few security settings are stored in the global cluster metadata as
+persistent settings. These are included in the snapshot by default, but for
+facilitating change reviews as well as restore convenience, we can pull them
+out to stand in their own file.
+
+[source,shell]
+--------------------------------------------------
+curl -X GET -u elastic "localhost:9200/_cluster/settings?pretty&flat_settings" | jq '.persistent | with_entries(select(.key|startswith("xpack.security")))'
+--------------------------------------------------
+
+You can store the output of this together with the other configuration files.
+
+NOTE: Transient settings are not considered for backup.
+
+--
+
+[float]
+[[backup-security-repos]]
+===== Controlling access to the backup repository
+
+If a third party reads the snapshot and exposes the contents of the `.security`
+index, it does not mean you have compromised cluster security. An attacker
```

Comment:
That's a broad statement to make.

Depending on the type of users that are stored, you may have regulatory reporting requirements if such a backup was exposed, even though it only has hashed passwords.

I think we need to tone down this wording.

I would have something along the lines of 

>The snapshot of the security index will typically contain sensitive data such
> as user names and password hashes. 
> Because password are stored using cryptographic hashes _(link to hashing config)_
> the disclosure of a snapshot would not automaticaly allow a third party 
> to authenticate as one of your users, but it would disclose confidential information.
> We recommend that you ...   
>
> It is also important that you protected the integrity of these backups in case you
> ever need to restore them. If a third party is able to modify the stored backups
> they may be able to install a back door that would grant access if the snapshot
> is loaded into an {es} cluster.